smart_card_connector_app/src/window.css

```diff
@@ -133,22 +133,26 @@ body {
   align-items: center;
 }
 
-.error-indicator {
+.status-indicator {
   width: 9px;
   height: 9px;
   border-radius: 50%;
   display: flex;
```

Comment:
I'm going to remove these "flex" declarations, as, first, I don't understand how they work and how to adopt them for more complex cases, and, second, because they seem to adjust the font size too (while the font sizes were carefully chosen previously together with the UI team).